tests/msbuild/Xamarin.MacDev.Tasks.Tests/TaskTests/LocalizationStringTest.cs

```diff
@@ -83,5 +84,77 @@ private string TranslateError (string culture, string errorCode)
 			PropertyInfo propertyInfo = typeof (MSBStrings).GetProperty (errorCode);
 			return (string) propertyInfo.GetValue (null, null);
 		}
+
+		[TestCase ("cs-CZ")]
+		[TestCase ("de-DE")]
+		[TestCase ("es-ES")]
+		[TestCase ("fr-FR")]
+		[TestCase ("it-IT")]
+		[TestCase ("ja-JP")]
+		[TestCase ("ko-KR")]
+		[TestCase ("pl-PL")]
+		[TestCase ("pt-BR")]
+		[TestCase ("ru-RU")]
+		[TestCase ("tr-TR")]
+		[TestCase ("zh-CN")]
+		[TestCase ("zh-TW")]
+		public void AllErrorTranslation (string culture)
+		{
+			string errorCode = string.Empty;
+			string errorList = string.Empty;
+			CultureInfo originalCulture = Thread.CurrentThread.CurrentUICulture;
+			string path = $"{Directory.GetCurrentDirectory ()}/TaskTests/LocalizationIgnore";
+
+			List<string> commonIgnoreList = ReadFile ($"{path}/common-Translations.ignore");
+			List<string> cultureIgnoreList = ReadFile ($"{path}/{culture}-Translations.ignore");
```

Comment:
This two lists can be a Set that is the resulting union of the ignored errors in the common file and in the current translation. What is more, there is no need to read the common ignore list for each of the cultures, we should do this once and re-use the value.